net.krafting.PleasureDVR.yml

```diff
@@ -0,0 +1,36 @@
+app-id: net.krafting.PleasureDVR
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: PleasureDVR.py
+finish-args:
+  -  --share=ipc
+  -  --share=network
+  -  --socket=wayland
+  -  --socket=fallback-x11
+  -  --filesystem=xdg-videos
+  -  --talk-name=org.freedesktop.Notifications
```

Comment:
is this really needed? Gtk does handle it properly usually.

Reply:
Does not seems to be needed indeed, will remove them.